packages/blocks/src/code-block/components/syntax-code-block.ts

```diff
@@ -9,10 +9,25 @@ const Module = Quill.import('core/module');
 const CodeBlock = Quill.import('formats/code-block');
 const CodeToken = Quill.import('modules/syntax');
 
+function addLineNumber(
+  container: HTMLElement,
+  lineHeight: string,
+  i: number | null
+) {
+  const node = document.createElement('div');
+  if (i) {
+    node.innerHTML = `${i}`;
```

Comment:
For safety reasons, use `textContent` please.

Reply:
Thanks, I will change this